### DISCUSSION TITLE
IRC GPT Bot cant connect in irc network

### DISCUSSION THREAD

**Discussion by imitev36** [knrd1/chatgpt#27] [General]
Hi knrd1 I have a problem with the bot it can't connect to network. The message it shows is this:
Connecting to: irc.anyserver.org
Connected to: irc.anyserver.org
NOTICE AUTH :*** Processing connection to irc.sofia.bg

NOTICE AUTH :*** Looking up your hostname...
NOTICE AUTH :*** Found your hostname
PING :47920E40
:irc.anyserver.org 451 * :You have not registered

I guess the code to return the PONG to the server needs to be fixed.
Please help me with this problem.

**Comment by knrd1** [comment]
Is irc.anyserver.org a valid IRC network? Can I have an exact IRC network address please so I can test it?

**Reply by cleobuline** [reply]
test it  on irc://labynet.fr
